src/app/core/guard/qbo-token.guard.ts

```
@@ -29,30 +29,35 @@
-            this.toastService.displayToastMessage(ToastSeverity.ERROR, 'Oops! Your QuickBooks Online connection expired, please connect again');
 
             const onboardingState: QBOOnboardingState = this.workspaceService.getOnboardingState();
-
             if (onboardingState !== QBOOnboardingState.COMPLETE) {
               return this.router.navigateByUrl('integrations/qbo/onboarding/connector');
             }
 
+            if (error.error.message === "Quickbooks Online connection expired"){
+              return this.router.navigateByUrl('integrations/qbo/token_expired/dashboard');
+            }
+            
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)